src/hotspot/share/opto/vectornode.hpp

```diff
@@ -569,6 +609,14 @@ class MaxVNode : public VectorNode {
   virtual int Opcode() const;
 };
 
+class UMaxVNode : public VectorNode {
+ public:
+  UMaxVNode(Node* in1, Node* in2, const TypeVect* vt) : VectorNode(in1,in2,vt) {
```

Comment:
```suggestion
  UMaxVNode(Node* in1, Node* in2, const TypeVect* vt) : VectorNode(in1, in2, vt) {
```
spaces required by style guide